src/util/serialize.h

```diff
@@ -267,6 +267,14 @@ inline v3f readV3F1000(const u8 *data)
 	return p;
 }
 
+inline v2f readV2F32(const u8 *data)
+{
+	v2f p;
+	p.X = (float)readF32(&data[0]);
+	p.Y = (float)readF32(&data[4]);
```

Comment:
Would it be possible to use `static_cast<float>` instead?

Reply:
copypasta. It's a redundant cast anyway.

Reply:
I'm wondering why to cast at all. `v2f` is made of `f32`'s, and `readF32` returns `f32`.